modules/3d-tiles/src/lib/parsers/parse-3d-tile-header.js

```diff
@@ -33,7 +33,13 @@ function getRefine(refine) {
 
 export function normalizeTileData(tile, options) {
   if (tile.content) {
-    tile.contentUrl = `${options.basePath}/${tile.content.uri}`;
+    let contentUri = tile.content.uri;
+    if ('url' in tile.content) {
+      // eslint-disable-next-line
+      console.warn('Tileset 3D: "content.url" property deprecated. Use "content.uri" instead.');
```

Comment:
Who is the target audience of this warning? Is this an issue in the metatdata file?

Reply:
My understanding is that this parser is not directly called by the user. What is the actionable fix for this warning?

Reply:
I think Cesium code base has such warnings, usually tileset creators test their tilesets there so it may make sense for them to issue this warning, but I agree it is not our job to warn about this. 

I guess it does flag that the tileset is older and that may be an early indicator of other issues.

Reply:
Old version of 3d tiles use `content.url`, but newer versions of tiles use `content.uri`. 
The intention is compatible with old versions of tiles.

Reply:
We can be compatible without warning.

Reply:
Especially the part where we tell the user to change the tileset is a bit much. Warning that it is an older tileset is probably fine.